config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_withExplicitProdProfile.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: sw.kogito.kie.org/v1alpha08
+kind: KogitoServerlessWorkflow
+metadata:
+  name: greeting
+  annotations:
+    sw.kogito.kie.org/description: Greeting example on k8s!
+    sw.kogito.kie.org/version: 0.0.1
+    sw.kogito.kie.org/profile: prod #It is not mandatory to add this annotation to use the Prod Profile
```

Comment:
I believe it is the default profile, right?
It would be nice to add it in the  comment as well

Reply:
I added this detail